test/shelf.test.js

```diff
@@ -97,7 +97,7 @@ describe('Generate usecase self-documentation', () => {
 
 		it('should generate some doc', () => {
 			//given
-			usecases.push({ usecase: givenTheSimplestUseCaseWithRequestResponse(), tags: { group: 'SimplestestUseCaseWithRequestResponse' } })
+			usecases.push({ id: 'AUsecaseWithResponse', usecase: givenTheSimplestUseCaseWithRequestResponse(), tags: { group: 'SimplestestUseCaseWithRequestResponse' } })
```

Comment:
this string is being repeated along this file, may be interesting create a const for it

Reply:
I think it's ok! The `usecases` variable is declared in different contexts - describe.